app/src/androidTest/java/com/github/sdp/ratemyepfl/dependencyinjection/HiltUtils.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.sdp.ratemyepfl.dependencyinjection
+
+import android.content.ComponentName
+import android.content.Intent
+import android.os.Bundle
+import androidx.core.util.Preconditions
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentFactory
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import com.github.sdp.ratemyepfl.activity.HiltTestActivity
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+object HiltUtils {
+    @ExperimentalCoroutinesApi
+    inline fun <reified T : Fragment> launchFragmentInHiltContainer(
```

Comment:
Yes, I'll do it. It basically does what launchFragmentInContainer does.

Reply:
Are you sure it's a good idea to use Experimental features in our code ?

Reply:
There is no other way, otherwise there would be some library to do it (and I would have done it). Note that it only affects a testing method, and if it happens to be deprecated someday we would only have to change this method (which is lighter than any other solution)

Reply:
I see, great !